storage/replica_command.go

```diff
@@ -1532,6 +1544,15 @@ func (r *Replica) VerifyChecksum(batch engine.Engine, ms *engine.MVCCStats, h ro
 		if p := r.store.ctx.TestingKnobs.BadChecksumPanic; p != nil {
 			p()
 		} else {
+			// About to panic print which k:v entries were different.
+			debug := args.Debug
+			for i := range debug.Entries {
+				e := debug.Entries[i]
+				v := c.debug.Entries[i]
+				if !bytes.Equal(e.Key, v.Key) || !bytes.Equal(e.Value, v.Value) {
```

Comment:
The output is going to be very verbose when the keys don't match. Might be better to use something like `github.com/kr/pretty.Diff` here which should output a minimal difference, though I'm not sure if we're linking that library into the production binary yet.